cmd/ack-generate/command/olm.go

```diff
@@ -0,0 +1,167 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/ghodss/yaml"
+
+	generate "github.com/aws-controllers-k8s/code-generator/pkg/generate"
+	ackgenerate "github.com/aws-controllers-k8s/code-generator/pkg/generate/ack"
+	olmgenerate "github.com/aws-controllers-k8s/code-generator/pkg/generate/olm"
+	ackmodel "github.com/aws-controllers-k8s/code-generator/pkg/model"
+	"github.com/spf13/cobra"
```

Comment:
femto-nit: I generally keep the third-party imports together separated by a newline and then the "local" imports, so sweep this cobra import up under the ghodss/yaml import line :)

Reply:
Fixed!